recipes/prismatic/meta.yaml

```diff
@@ -0,0 +1,173 @@
+{% set version = "1.2.1" %}
+{% set build = 0 %}
+
+{% set processor = processor or 'cpu' %}
+
+{% if processor == "cpu" %}
+# prioritize CPU version via build number
+{% set build = build + 100 %}
+{% endif %}
```

Comment:
As mentioned by @isuruf in https://github.com/conda-forge/ucx-split-feedstock/issues/26#issuecomment-543313958, would it be possible to use "run_constrained on __cuda for GPU builds" instead of setting a default priority on the cpu build with higher build number?

Reply:
Possibly. It would make more sense to users - I'll do some reading into it, to see how it works.

Reply:
I've removed the code bumping the version number for the moment - as we're currently building only for CPU.